.github/workflows/test.yml

```diff
@@ -55,6 +51,16 @@ jobs:
           repository: "${{ needs.configure.outputs.repo-name }}"
           persist-credentials: false
 
+      - name: Check if change in /operators or in any .go file
+        uses: dorny/paths-filter@v2
+        id: pathFilter
+        with:
+          filters: |
+            go:
+              - '**.go'
+            operators:
+              - 'operators/**'
```

Comment:
The operators filter should be enough, as we do not have go files outside that folder (and they would not be tested anyhow).